packages/ckeditor5-link/tests/linkimageediting.js

```diff
@@ -826,6 +826,34 @@ describe( 'LinkImageEditing', () => {
 					'</p>'
 				);
 			} );
+
+			it( 'should downcast without error if the image already has no link', () => {
```

Comment:
I'd add two more tests. In the one you adde you check whether what the command does works. But what if someone operates on the attributes themselves? Then they can do things in a different order and also they can mix those operations with other, potentially conflicting operations (e.g. set decorator attr, remove linkHref). I actually don't know what will happen in this case in the model as I don't know if there's some postfixer that cleans up decorator attrs.

Reply:
Just added another test for reversed order of downcasters. Will the first test be enough to cover the "regular" order, or should I explicitly add it?

Reply:
Actually, it makes sense to add another test with an explicit, "default" order of operations. Just in case the default behavior changes.